Edits to chapters 1-3 (#122)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-to-non-standard-network-topologies.adoc

// TODO: remove this after OSP13 z13 since it will no longer be necessary.

> If you use {ProjectShort} with {OpenStack} {OpenStackVersion}  and plan to monitor your Ceph, Block, or Object storage nodes, you must make a similar adjustment to what is done for spine-leaf and DCN network configuration. To monitor Ceph nodes, use the `CephStorageExtraConfig` parameter to define which network interface to load into the {MessageBus} and collectd configuration files.

Reviewer: Please check that this is accurate first, but to me, 'making adjustments' sounds too colloquial in this case.
```suggestion
If you use {ProjectShort} with {OpenStack} {OpenStackVersion}  and plan to monitor your Ceph, Block, or Object storage nodes, you must make configuration changes that are similar to the configuration changes that you make to the spine-leaf and DCN network configuration. To monitor Ceph nodes, use the `CephStorageExtraConfig` parameter to define which network interface to load into the {MessageBus} and collectd configuration files.
```
